source/delegate/test/Delegate.js

```diff
@@ -473,20 +466,6 @@ contract('Delegate Integration Tests', async accounts => {
         },
       })
 
-      // authorize the delegate to be sender
-      let expiry = await getTimestampPlusDays(0.5)
-      let tx = await swapContract.authorizeSender(
-        aliceDelegate.address,
-        expiry,
-        {
-          from: aliceTradeWallet,
-        }
-      )
-      emitted(tx, 'AuthorizeSender')
-
-      // increase time past expiry
-      await advanceTime(SECONDS_IN_DAY * 0.6)
-
       // Succeeds on the Delegate, fails on the Swap.
       // aliceTradeWallet approval has expired
```

Comment:
This test is now identical to the one above it. They both:
- create an order
- submit it from an unauthorized sender

The name of the second test says 'revoked', so suggest this second test (that used to increase time past expiry), now:
- create an order
- authorized sender
- revoke authorization
- try to send (unauthorized)